azext_iot/sdk/digitaltwins/controlplane/models/connection_properties_private_link_service_connection_state_py3.py

```diff
@@ -13,14 +13,14 @@
 
 
 class ConnectionPropertiesPrivateLinkServiceConnectionState(ConnectionState):
-    """ConnectionPropertiesPrivateLinkServiceConnectionState.
+    """The connection state.
 
     All required parameters must be populated in order to send to Azure.
 
     :param status: Required. The status of a private endpoint connection.
      Possible values include: 'Pending', 'Approved', 'Rejected', 'Disconnected'
     :type status: str or
-     ~controlplane.models.PrivateLinkServiceConnectionStatus
+     ~azure.mgmt.digitaltwins.models.PrivateLinkServiceConnectionStatus
```

Comment:
Why describe a different root namespace? It's technically no longer accurate.

Reply:
got generated like that - can manually fix

Reply:
Ok. At least in prior autorest configurations you could specify the namespace.